core/workspace_svg.js

```diff
@@ -54,6 +54,11 @@ goog.require('goog.math.Coordinate');
 goog.require('goog.userAgent');
 goog.require('goog.math.Rect');
 
+/**
+ * This is a unique object that only we have access to.  It is used
```

Comment:
This is a global declaration.  

This also will causes problems with multiple instances of Blockly on the same page.

Reply:
oh ... hrm, THANKS! -- I'm so used to modules...

should I make it `Blockly.WorkspaceSVG.DIRTY` ? I just wanted a unique value that wasn't `null` - it could even be `false`

Reply:
Oh right, won't actually cause problems because you're not using the fact that it's an object to store anything.

Yeah either put it on Blockly.WorkspaceSvg or put it in Blockly.Constants.  I would probably just use the string "dirty" instead of an empty object.

Reply:
ooh - string `"dirty"` is even better than false ❤️